src/components/SocialSharing/index.tsx

```diff
@@ -5,37 +5,56 @@ import {
   FacebookShareButton,
   FacebookIcon,
   TwitterShareButton,
-  TwitterIcon,
+  XIcon,
+  WhatsappShareButton,
+  WhatsappIcon,
+  EmailShareButton,
 } from 'react-share';
+import Image from 'next/image';
+
+const mailIcon = '/images/mail_icon.svg';
 
 const StyledContainer = styled.div`
   display: flex;
-  align-items: center;
+  flex-direction: column;
   margin: 20px 0;
   span {
     font-size: ${p => p.theme.fontSizeL};
   }
 `;
 
 const ButtonsContainer = styled.div`
-  margin-left: 10px;
+  margin-top: 15px;
+  display: flex;
+  justify-content: flex-start;
 
   > *:not(:first-child) {
-    margin-left: 5px;
+    margin-left: 10px;
   }
 `;
 
 const SocialSharing: FC = () => {
   return (
     <StyledContainer>
-      <span>Teilen:</span>
+      <span>Teile Gieß den Kiez mit deinem Umfeld und hilf uns die Gieß-Community zu vergrößern:</span>
       <ButtonsContainer>
         <FacebookShareButton url='https://www.giessdenkiez.de/'>
           <FacebookIcon size={36} round />
         </FacebookShareButton>
         <TwitterShareButton url='https://www.giessdenkiez.de/'>
-          <TwitterIcon size={36} round />
+          <XIcon size={36} round />
         </TwitterShareButton>
+        <WhatsappShareButton url='https://www.giessdenkiez.de/'>
+          <WhatsappIcon size={36} round />
+        </WhatsappShareButton>
+        <EmailShareButton url='https://www.giessdenkiez.de/'>
+        <Image
+                src={mailIcon}
+                width={36}
+                height={36}
+                alt='Mail Icon'
+              />
```

Comment:
Indentation and line wrapping is wrong here in various places - I suspect that you don't have enabled "format on save" with `prettier`?